partner_tlr_update/models/res_partner.py

```diff
@@ -200,11 +200,13 @@ def update_contact_person(self, xml, parent_id):
         if person_id is not None:
             person_id = person_id.text
             children = self.env['res.users'].search([
-                ('parent_id', '=', parent_id),
-                ('legacy_no', '=', person_id)], limit=1)
+                ('legacy_no', '=', person_id)], limit=1)   
             if children:
                 _logger.info("found children contact persons")
                 user = children
+                if children.parent_id.id != parent_id:
+                    _logger.info("found contact person with different parent_id, overwriting")
```

Comment:
Row to long.
As it is within () it's easy to split it up:
("found contact person with different "
 "parent_id, overwriting")